docs/pages/build/submitting-to-app-stores.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Submitting builds to App Stores
+---
+
+This guide outlines how to submit your app to Apple App Store or Google Play Store. A paid developer account is required for each platform for which you wishy to submit an app. You can create an Apple Developer account on [Apple's developer site](https://developer.apple.com/account/) and a Google Play Developer account on the [Google Play Console sign-up page](https://play.google.com/apps/publish/signup/).
+
+## 1. Build a standalone app
+
+You'll need a built native app binary. You can either use [EAS Build](introduction.md) service or do it on your own.
+
+## 2. Start the submission
+
+To submit the previously built app to the appropriate app store, you simply run `eas submit -p ios` or `eas submit -p android` from inside of your project directory. Except specifying platform, no other flags are required, the command will lead you step by step. To see all possible options along with example usages, run `eas submit --help`.
```

Comment:
```suggestion
To submit the previously built app to the appropriate app store, run `eas submit -p ios` or `eas submit -p android` from inside of your project directory. The command will lead you step by step through the process of submitting the app. To see all possible options along with example usages, run `eas submit --help`.
```

also, we should probably not make the flag required. if a user runs `eas submit` without a platform flag we should let them choose between ios/android, using this style of prompt.

![Screen Shot 2020-11-30 at 5 27 40 PM](https://user-images.githubusercontent.com/90494/100685507-607ab280-3331-11eb-8a43-d8241214f57b.png)

Reply:
Good idea, added to my list. Currently this flag is required to be consistent with `eas build:create` which also requires the flag